test/WebSites/ValidationWebSite/CompanyNameAttribute.cs

```diff
@@ -3,19 +3,16 @@
 
 using System.ComponentModel.DataAnnotations;
 
-namespace FormatterWebSite
+namespace ValidationWebSite
 {
-    public class AdminValidator : ValidationAttribute
+    public class CompanyNameAttribute : ValidationAttribute
     {
         protected override ValidationResult IsValid(object value, ValidationContext validationContext)
         {
-            var admin = (Administrator)value;
-            
-            if (admin.AdminAccessCode != 1)
+            if (value == null)
```

Comment:
to test if `value` is a non-`null` `string`, use two statements e.g. `var valueString = value as string; if (string.IsNullOrEmpty(valueString) ...`